apps/app/pages/index.tsx

```diff
@@ -79,8 +99,71 @@ function Dashboard() {
                 title="Tens a certeza que queres notificar?"
                 cancelText="Não"
                 okText="Sim"
-                onConfirm={(_) => notify_signup_ninjas()}
+                onConfirm={(_) => {
+                  notify_signup_ninjas();
+                  setConfirmed_signup(true);
+                }}
+                disabled={confirmed_signup}
               >
+                <Modal
+                  visible={visible_signup}
+                  onCancel={handleCloseModal_signup}
+                  centered={true}
+                  closable={false}
+                  title="Lista de Emails"
+                  footer={[
+                    <Button
+                      key="close"
+                      type="primary"
+                      onClick={handleCloseModal_signup}
+                    >
+                      Close
+                    </Button>,
+                  ]}
+                >
+                  <div
+                    style={{ display: "flex", justifyContent: "space-between" }}
+                  >
+                    <div
+                      style={{
+                        width: "45%",
+                        height: "500px",
+                        overflowY: "scroll",
+                        overflowX: "scroll",
+                      }}
+                    >
+                      <div
+                        style={{ display: "flex", justifyContent: "center" }}
+                      >
+                        <Button type="primary">Entregues</Button>
+                      </div>
+                      <ul>
+                        {emails_sucess.map((email, index) => (
+                          <li key={index}>{email}</li>
+                        ))}
+                      </ul>
+                    </div>
+                    <div
+                      style={{
+                        width: "45%",
+                        height: "500px",
+                        overflowY: "scroll",
+                        overflowX: "scroll",
+                      }}
+                    >
+                      <div
+                        style={{ display: "flex", justifyContent: "center" }}
+                      >
+                        <Button type="primary">Entregues</Button>
```

Comment:
This button should display "Não entregues". Also, I don't like its style. I would suggest you to take a loot at [this](https://ant.design/components/alertl) component from ant design. 
A divider (there's a component from ant design) between the buttons and the emails would look awesome too 👀